src/ghazal_azadi_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,34 @@
+def letter_freq(txt):
+    txt = txt.lower()
+    letter_new = {}
+    for i in txt:
+        letter_new[i] = txt.count(i)
+    return letter_new
+
+
+def entropy(message):
+    # Function which calculates the entropy of the given msg.
+    import math
+    p = []
+    freq = []
+    number = len(message)
+    n = list(letter_freq(message).values())
```

Comment:
No we don't need to convert it to list. (Doing that makes the use of letter_freq function meaningless.